### DISCUSSION TITLE
Xiaomi MCCGQ01LM Contact Sensor paired but no data

### DISCUSSION THREAD

**Discussion by DeckardCain2014** [Koenkk/zigbee2mqtt#20864] [Devices]
Hi,
my contact sensor recently dropped its connection to my gateway. I'm using a Sonoff Zigbee stick and a lot of IKEA bulbs as routers. I switched the battery to a brand new one. Now it is successfully paired, but shows no data (all values N/A).
Is it broken or can I fix it in any way?

**Comment by mr2c12** [comment]
Just happened to me.
A sensor I have been using for 3 years just dropped out of the network. Sometimes it happens, and usually it's quite easy to have it rejoined.

Today, something completely different.
If I repair from its position (using a Xiaomi smart socket as reference router for pairing), the pairing process completes but "contact" is "null" or "N/A".
If I repair the sensor very close to the coordinator, I can see "contact" "open" but I don't get any update when expected. If I got it back to its original position, I can't get any update, too - but this is quite expected, as the coordinator is very far.
